lib/eval_test.js

```diff
@@ -116,6 +116,7 @@ function composeNormalizer(normalizers) {
  * status of this test (whether it passed, failed, is a placeholder, etc.)
  */
 function evalTest( testCase, apiResults, context ){
+  context = _.clone(context);
```

Comment:
I'd either pass `true` for the `deep` param, call `cloneDeep` instead to ensure you have the full copy you think you do.